src/main/java/racingcar/ui/InputView.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.ui;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class InputView {
+    private static InputView instance = new InputView();
```

Comment:
```suggestion
    private static final InputView instance = new InputView();
```
런타임 때 변경되지 않도록 final을 붙여주면 더 좋겠습니다.

성현님이 구현하신 싱글톤 클래스 모두에게 해당되겠네요